src/clr-addons/quick-list/quick-list.component.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'clr-quick-list',
+  templateUrl: './quick-list.component.html',
+  styleUrls: ['./quick-list.component.scss'],
+})
+export class ClrQuickListComponent implements OnInit {
+  @Input() label: string;
```

Comment:
According to our coding guidelines, we try to be conform with clarity and every public class/property needs to be prefixed with "clr" (see e.g. combobox).
Contribution Guidelines: https://confluence.porscheinformatik.com/confluence/pages/viewpage.action?spaceKey=PSACNG&title=Contribution+to+Clarity-Addons
Coding Guidelines: https://github.com/vmware/clarity/blob/master/CODING_GUIDELINES.md